west.yml

```diff
@@ -100,6 +102,10 @@ manifest:
     - name: littlefs
       path: modules/fs/littlefs
       revision: fe9572dd5a9fcf93a249daa4233012692bd2881d
+    - name: public-mipi-sys-t
```

Comment:
please use github.com:zephyrproject-rtos/mipi-sys-t.git instead and put this in its own commit.

Reply:
ok

Reply:
@nashif have already used zephyrproject-rtos/mipi-sys-t.git.